Source/TractUI.cpp

```diff
@@ -196,12 +295,11 @@ void TractUI::moveTo(Graphics &g, t_tractProps *props, Path &p, double i, double
 {
 	double angle = this->angleOffset + i * this->angleScale * M_PI / (props->lipStart - 1);
 	
-	// TODO: Figure out wobble
-//	double time = 0.0;
-//	double wobble = (Tract.maxAmplitude[Tract.n-1]+Tract.noseMaxAmplitude[Tract.noseLength-1]);
-//	wobble *= 0.03 sin(2 * i - 50 * time) * i / (double) props->n;
-//	angle += wobble;
-	double wobble = 0.0;
+    time_t t = time(NULL);
```

Comment:
We really, really shouldn't be grabbing time in a function called "moveTo". Maybe pass the time as an argument instead? Also it looks like we don't actually need the true time. We've got a timer that fires 60 times a second, so we could just increment an internal millisecond counter by 1000/60 milliseconds every time the timer fires. Much more efficient than calling mktime in this function.